lib/full-site-editing/block-templates.php

```diff
@@ -367,6 +371,25 @@ function gutenberg_get_block_template( $id, $template_type = 'wp_template' ) {
 		}
 	}
 
+	return gutenberg_get_block_file_template( $id, $template_type );
+}
+
+/**
+ * Retrieves a single unified template object using its id.
+ * Retrieves the file template.
+ *
+ * @param string $id Template unique identifier (example: theme|slug).
```

Comment:
Let's update this description with the appropriate ID format!

```suggestion
 * @param string $id Template unique identifier (example: theme_slug//template_slug).
```